.env.example

```diff
@@ -0,0 +1,10 @@
+####################
+### EXAMPLE FILE ###
+####################
+
+# Do NOT edit this file unless adding new env variables
+# Instead, copy it, remove the .example extension and provide your custom values
+
+WINDUP_PATH=path/to/windup-cli
+OUTPUT_PATH=path/to/export/the/report/in/
+PROJECT_PATH=path/to/project
```

Comment:
and this to "input_application_path"

as this is the input parameter we provide in CLI

Reply:
That is not the input we provide in CLI.

Is te absolute path to the project (the folder where you have this project in local), which is needed to finde the binary files stored in the project and have access to them without requiring to use sudo.